CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [1.1.3] - 2022-07-11
```

Comment:
No need to add this; our release workflow fills in the changelog when we press the "release" button.